include/picongpu/fields/absorber/pml/Field.tpp

```diff
@@ -124,32 +124,37 @@ namespace picongpu
                     // Note: since this should compile for 2d, .z( ) can't be used
                     using detail::makeIdx;
                     int layerIdx = 0;
-                    if constexpr(simDim == 3)
+
+                    auto positiveBeginZ = 0;
```

Comment:
Please put comment here. Along the lines of
```
Define standard values of Layer z-origin and negative z-size for 2D simulations.
```

Reply:
done